### PR TITLE
Fix-63 proposals design fixes

### LINKED ISSUE #63: Updated list of proposals according according to the designs
## Description
There is a number of cosmetic bugs with listing component to be addressed bundled in the same issue, since they are all quite small

- [ ] Remove proposals IDs;
- [ ] Change table header font to Founder Grotesk Medium (32)
- [ ] Remove a line at the beginning of voting bars:

- [ ] All statuses should be written in black
- [ ] The width of a status pill should be 100, not 95

[Component in Figma
](https://www.figma.com/file/QSQsvksBOspBTONiBBFpqa/MENTO-Airgrab?type=design&node-id=6395%3A8169&mode=design&t=hhJPILyjWbCu1QPx-1)
## Acceptance criteria
- [ ] All points from the list above are addressed

### PR DESCRIPTION
**Pull request by AnnaJ-P11**
### Description

- the entire component was checked and changes were made in line with the designs
-  fixed progressbar styling when the value is minimal 
-  fix progressbar styling
- Remove proposals IDs
- Change fonts in whole proposals list
- change statuses to be written in black

### Other changes

- add Inter font

### Tested

Go to porposals list

### Related issues

-  Fixes #63

### CODE REVIEW

File: app/components/_shared/progress-bar/progress-bar.module.scss [hunk 1]
```diff
@@ -5,13 +5,18 @@
   flex-direction: column;
   align-items: flex-end;
   justify-content: center;
-
+  text-align: right;
+  font-family: "Founders Grotesk";
```
Comment (tplocic20) [inline app/components/_shared/progress-bar/progress-bar.module.scss:9]: I would move the font definition to the global file

File: app/globals.scss [hunk 2]
```diff
@@ -44,6 +44,70 @@ body {
   font-style: normal;
 }
 
+@font-face {
```
Comment (tplocic20) [inline app/globals.scss:47]: Shouldn't we be using the Founders Grotesk globally?
So we probably don't need the Inter font

**Review by tplocic20 — CHANGES_REQUESTED** [review]

**Review by tplocic20 — APPROVED** [review]

File: app/components/proposals-list/proposals-list.component.tsx [hunk 3]
```diff
@@ -77,22 +71,10 @@ export const ProposalsListComponent = ({
...
-                    className={classNames(
-                      styles.status,
-                      "uppercase font-medium",
-                    )}
-                    type={stateToBadgeColorMap[state]}
-                  >
-                    {state?.toString()}
-                  </Badge>
-                </div>
+                <Status
```
Comment (denviljclarke) [inline app/components/proposals-list/proposals-list.component.tsx:74]: The color for the 'Active' status doesn't seem correct.

File: app/components/proposals-list/proposals-list.component.tsx [hunk 4]
```diff
@@ -77,22 +71,10 @@ export const ProposalsListComponent = ({
...
+                  text={state?.toString()}
+                  type={stateToBadgeColorMap[state]}
+                />
                 <div
```
Comment (denviljclarke) [inline app/components/proposals-list/proposals-list.component.tsx:78]: The min gap between the progress bar and the title is a bit too small. It's larger in the designs

File: app/components/_shared/progress-bar/progress-bar.component.tsx [hunk 5]
```diff
@@ -80,7 +126,11 @@ export const MultiProgressBar = ({
             <div
```
Comment (denviljclarke) [inline app/components/_shared/progress-bar/progress-bar.component.tsx:126]: Something is causing there to be a small black element between the end of the bar and the container

**Comment by denviljclarke** [comment]
Nice @AnnaJ-P11. I left a few comments. I also made a commit adding inter using Next.js. I think it's good to use it from there since we already have it installed. 

Also, I see in the designs that there is supposed to be pagination this component, do you know why it's not implemented? 

[Link to pagination in the design](https://www.figma.com/file/QSQsvksBOspBTONiBBFpqa/MENTO-Airgrab?type=design&node-id=8669-3746&mode=design&t=hCDHbnqdmTMkuWFW-4)

**Comment by AnnaJ-P11** [comment]
> Nice @AnnaJ-P11. I left a few comments. I also made a commit adding inter using Next.js. I think it's good to use it from there since we already have it installed.
> 
> Also, I see in the designs that there is supposed to be pagination this component, do you know why it's not implemented?
> 
> [Link to pagination in the design](https://www.figma.com/file/QSQsvksBOspBTONiBBFpqa/MENTO-Airgrab?type=design&node-id=8669-3746&mode=design&t=hCDHbnqdmTMkuWFW-4)

Ok, thanks. It looks like pagination has recently been added to the design. I haven't seen it before

**Review by denviljclarke — APPROVED** [review]
I [removed some code](https://github.com/mento-protocol/governance-ui/pull/78/commits/6fc21169b9467b92d7fe98736b9b01414fade0b8) which I introduced in error when testing how a shorter proposal list would look. 

Apart from that note - LGTM 👍

**Pull request merged**